src/core/CMakeLists.txt

```diff
@@ -863,8 +863,37 @@ target_link_libraries(inviwo-core
         utf8::cpp
 )
 
-#Optimize compiliation with PCH etc
-ivw_compile_optimize_on_target(inviwo-core)
+set(PCH_HEADERS
```

Comment:
Would be nice with a comment on how to reason when adding/removing headers here

Reply:
Yes, what about adding vector? unordered_(map/set) 
Why are you adding c headers?

Reply:
These are taken from the top list of includes taking the most time as measured with the CompileScore extension.

Reply:
what happens if you add more?

Reply:
There seems to be no impact. Tried with `data.h`, `volume.h`, and `image.h` with no effect even though they accounted for quite some include times. Needs testing with additional standard headers.